src/App.js

```diff
@@ -441,7 +441,7 @@ class App extends Component {
           console.log(ex, 'Gaia put exception');
         })
     } else {
-      const prefs = JSON.parse(localStorage.getItem("pref"))
+      const prefs = JSON.parse(localStorage.getItem("pref")) || {};
```

Comment:
@williamluke4 had to make one small change to the default here, as it fails for a user without a preference already set to local storage. For clarity in case you were wondering, th `if (isUserSignedIn())` logic just above (and all the logged in areas) are for a spinoff at http://coinkitty.co using Blockstack distributed storage (instead of `localStorage`)